### PR TITLE
KEYCLOAK-17828 add enabled event types in CRD

### PR DESCRIPTION
**Pull request by alexandrelussier**
## JIRA ID

https://issues.redhat.com/browse/KEYCLOAK-17828
## Additional Information

This pull request adds the possibility to specify which events are enabled when EventsEnabled is true

1. Create a Realm with a few events in the enabled list (see the example in this PR)
2. Go to the admin portal, in the Realm you created
3. Go to the event config section
4. You should see only the events you put in the Realm definition (step 1)

## Checklist:
- [x] Automated Tests
- [x] [Documentation](https://github.com/keycloak/keycloak-documentation) changes if necessary
- [x] (If required) Discussed on [Keycloak DEV](https://groups.google.com/forum/#!forum/keycloak-dev)

### CODE REVIEW

**Comment by vmuzikar** [comment]
@alexandrelussier Thank you for your contribution! Unfortunately, I don't have the permissions to run the checks. @slaskawi Could you please do it?

**Comment by slaskawi** [comment]
@vmuzikar Let me assign this one to you.

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@slaskawi Thanks for approving the CI run!

@alexandrelussier It seems there's some test failure related to this change.

**Comment by alexandrelussier** [comment]
@vmuzikar sorry about the failing test, I cannot run the test on my machine even with a clean branch. I will try to fix that first.

**Comment by vmuzikar** [comment]
@slaskawi Can you please approve the CI?

**Comment by alexandrelussier** [comment]
Thanks for restarting the CI, but I wasn't ready, sorry.

@vmuzikar I just push a modification to my test and it is passing now, can you check?

**Comment by alexandrelussier** [comment]
@vmuzikar it should be good to go. Do you have time to review this?

File: go.mod [hunk 1]
```diff
@@ -11,6 +11,7 @@ require (
 	github.com/sirupsen/logrus v1.5.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
+	golang.org/x/tools v0.1.0 // indirect
```
Comment (vmuzikar) [inline go.mod:14]: I might be missing something but why are you adding this?

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@alexandrelussier Sorry for late reply and thanks for the update!

File: go.mod (hunk 1 above)
Comment (alexandrelussier) [reply go.mod:14]: good question, it was done automatically when I did make code/gen. I will revert that

**Review by vmuzikar — APPROVED** [review]
@alexandrelussier Thanks for the update! If the CI passes, I think we can merge.

**Review by abstractj — APPROVED** [review]

**Pull request merged**